neo4j-ruby-driver.gemspec

```diff
@@ -33,22 +36,24 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
 
-  ffi = ENV['SEABOLT_LIB']&.length&.positive?
-  pdir = ffi ? 'ffi' : 'jruby'
+  jruby = RUBY_PLATFORM.match?(/java/)
+  pdir = if ENV.key?('SEABOLT_LIB')
+           'ffi'
+         else
+           jruby ? 'jruby' : 'ffi'
+         end
 
   spec.files << Dir['lib/**/*.rb']
   spec.files << Dir["#{pdir}/**/*.rb"]
-  spec.files << Dir["#{pdir}/**/*.jar"]
 
   spec.bindir = 'exe'
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib', pdir]
 
-  spec.platform = 'java' if RUBY_PLATFORM.match?(/java/)
+  spec.add_runtime_dependency 'ffi'
 
-  if ffi
-    spec.add_runtime_dependency 'ffi'
-  else
+  if jruby
+    spec.platform = 'java'
     spec.add_runtime_dependency 'jar-dependencies'
```

Comment:
this is incorrect as well, we don't want to always include jar dependencies on jruby, ff/jruby does not need them

Reply:
Ok, but in this case if user will install gem with `SEABOLT_LIB` set and then will use it without that variable he will get an error about missing jars